.travis.yml

```diff
@@ -1,20 +1,39 @@
 language: python
+
 matrix:
-  include:
-    - python: 2.7
-      env:
-        - TOXENV=django19-py27
-        - TOXENV=django110-py27
-        - TOXENV=django111-py27
-    - python: 3.6
-      env:
-       - TOXENV=django19-py36
-       - TOXENV=django110-py36
-       - TOXENV=django111-py36
+    include:
+        - python: 2.7
```

Comment:
List these envs the same way please. No need for the separation the be this granular.